### PR TITLE
pluma-notebook.c: avoid deprecated 'gdk_pointer_ ...' functions

### PR DESCRIPTION
**Pull request by sc0w**
avoid deprecated:

gdk_pointer_is_grabbed
gdk_pointer_grab
gdk_pointer_ungrab

### CODE REVIEW

File: pluma/pluma-notebook.c [hunk 1]
```diff
@@ -396,31 +396,37 @@ static void
...
-				  GDK_BUTTON1_MOTION_MASK | GDK_BUTTON_RELEASE_MASK,
-				  NULL, 
-				  cursor, 
-				  time);
+		gdk_seat_grab (seat,
+			       gtk_widget_get_window (GTK_WIDGET (notebook)),
+			       GDK_SEAT_CAPABILITY_POINTER,
+			       FALSE,
+			       cursor,
+			       NULL,
```
Comment (monsta) [inline pluma/pluma-notebook.c]: Hmm, `time` argument in `drag_start` function is now unused. We can either drop it from the function, or replace it with a `GdkEvent`, which can be passed into `gdk_seat_grab`.

Comment (sc0w) [reply pluma/pluma-notebook.c]: ok, please, feel free to do the job :)

Comment (monsta) [reply pluma/pluma-notebook.c]: Sure, I'll make changes.

Comment (monsta) [reply pluma/pluma-notebook.c]: There we go, https://github.com/mate-desktop/pluma/pull/326

**Pull request merged**